django_seo_js/backends/test.py

```diff
@@ -25,7 +25,7 @@ def get_response_for_url(self, url):
         r["test rendered"] = "headers"
         r.status_code = 503
         r.content_type = "text/html"
-        assert r.status_code < 500
+        assert r.status_code == 503
```

Comment:
The test should not be deleted b/c it serves as a template for other test cases. We could delete the assert statement.